modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -22,17 +23,19 @@ In earlier versions of {product}, defining conditional policies with a policy ac
 
 This issue led to difficulties in defining conditional policies, hindering the configuration of permissions within the application.
 
-With this fix, you can now define conditional policies in {product-short} with permission policy actions of 'use'. link:{LinkRHIDPIssue}RHIDP-2237[(RHIDP-2237)]
+With this fix, you can now define conditional policies in {product-short} with permission policy actions of 'use'. 
+//link:{LinkRHIDPIssue}RHIDP-2237[RHIDP-2237]
 --
 
 Setting a custom route not working when using the {product-short} Operator on OpenShift::
 +
 --
 In earlier versions of the {product} Operator, setting a custom Route host on OpenShift Container Platform by using the `spec.application.route.host` field in the Custom Resource was not possible.
 
-    This limitation prevented users from configuring custom Route hosts, limiting their ability to customize the deployment environment.
+This limitation prevented users from configuring custom Route hosts, limiting their ability to customize the deployment environment.
```

Comment:
```suggestion
This limitation prevented users from configuring custom route hosts, limiting their ability to customize the deployment environment.
```
Similar to previous convos with Lindsey, I'd recommend using lowercase for general mentions of things like routes, and only capitalizing if we're referring to a specific "`Route` object", basically consistent with https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting